src/io.ts

```diff
@@ -1,32 +1,19 @@
 import { createHash } from 'crypto';
 import { join, parse } from 'path';
-import { promisify } from 'util';
-import { access } from 'fs';
-import { ensureDir, readdir, readJSON, unlink, writeJson } from 'fs-extra';
+import { ensureDir, existsSync, readdir, readJSON, unlink, writeJson } from 'fs-extra';
 import { buildTodoData } from './builders';
 import { FilePath, LintResult, TodoData } from './types';
 
-const exists = promisify(access);
-
-export async function todoDirExists(baseDir: string): Promise<boolean> {
-  try {
-    await exists(getTodoStorageDirPath(baseDir));
-    return true;
-  } catch (error) {
-    if (error.code === 'ENOENT') {
-      return false;
-    }
-
-    throw error;
-  }
+export function todoStorageDirExists(baseDir: string): boolean {
+  return existsSync(getTodoStorageDirPath(baseDir));
```

Comment:
This change is to avoid the potential issues with race conditions with an async exists.